pubpub/pubpub
# display large twitter card on pasted links if avatar image is set

## Issue(s) Resolved
#2424

## Test Plan
1. go to [review app](https://pubpub-pipel-bh-big-twi-tofxfx.herokuapp.com/)
2. find or create a pub with a header image
3. publish that pub
4. copy link to published pub
5. open any twitter account
6. paste the copied link into a reply, for example [on my testing thread](https://twitter.com/kfgdev/status/1641111333520154624?s=20)
7. link should display with full-width image
8. delete the post
9. remove the image from pub settings
10. repeat 3-6
11. image should be smaller and on left side

## Screenshots (if applicable)

Below in order, unset image behavior, current live behavior, and new behavior:
![Screenshot from 2023-03-29 14-54-55](https://user-images.githubusercontent.com/14115194/228639696-404709dc-1272-4203-99b7-9ecc9bd7f814.png)


## Optional

### Notes/Context/Gotchas
Doesn't implement the additional requests from #2424, which will live to see another milestone.

### Supporting Docs
